shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -681,19 +694,67 @@ - (void)setBaseWritingDirection:(UITextWritingDirection)writingDirection
 
 #pragma mark - UITextInput cursor, selection rect handling
 
+- (void)setMarkedRect:(CGRect)markedRect {
+  _markedRect = markedRect;
+  // Invalidate the cache.
+  _cachedFirstRect = kInvalidFirstRect;
+}
+
+- (void)setEditableTransform:(NSArray*)matrix {
```

Comment:
You should probably comment the matrix is row major since there are 2 ways to define matrices linearly